core/sawtooth/cli/admin.py

```diff
@@ -17,8 +17,8 @@
 from sawtooth.cli.exceptions import CliException
 from sawtooth.cli.admin_sub.dev_mode_genesis import add_dev_mode_genesis_parser
 from sawtooth.cli.admin_sub.dev_mode_genesis import do_dev_mode_genesis
-from sawtooth.cli.admin_sub.poet1_genesis import add_poet1_genesis_parser
-from sawtooth.cli.admin_sub.poet1_genesis import do_poet1_genesis
+from sawtooth.cli.admin_sub.poet1_genesis import add_poet_genesis_parser
```

Comment:
Yeah....it turns out that there is no poet1_genesis.py

I contemplated completely removing if you are okay with that.  Otherwise I was going to leave it and do a follow-up story/task to clean up that cruft.